eng/SharedFramework.Local.props

```diff
@@ -52,6 +52,7 @@
     <AspNetCoreAppReference Include="Microsoft.AspNetCore.Authentication.OAuth" />
     <AspNetCoreAppReference Include="Microsoft.AspNetCore.Authorization" />
     <AspNetCoreAppReference Include="Microsoft.AspNetCore.Authorization.Policy" />
+    <AspNetCoreAppReference Include="Microsoft.AspNetCore.Components.Server" />
```

Comment:
Looks like this duplicates a change in @pranavkm's PR https://github.com/aspnet/AspNetCore/pull/8970/files

@javiercn and @pranavkm - can you coordinate this, or are you already?

Reply:
Ah, I didn't see @pranavkm PR. I chatted with @danroth27 and decided to make it while I was here.

Reply:
I don't see how @pranavkm change adds it to the shared framework.

Reply:
OK, can you and @pranavkm work out which change is going in?